client/rest-high-level/src/test/java/org/elasticsearch/client/RestHighLevelClientTests.java

```diff
@@ -326,59 +322,6 @@ public void testParseEntity() throws IOException {
         }
     }
 
-    public void testParseCompressedEntity() throws IOException {
```

Comment:
these high level rest client decompression tests are still valid right?

Reply:
Well the decompression happens within the llrc when executing request and parsing the response. When the response is given to the rhlc the compressed data is already decompressed. In any kind of scenario the rhlc will always get a decompressed http enity, so therefor I can't unit test that method within the rhlc anymore.

What do you think, should we drop this unit test or do you think there is a way to validate this use case within the rhlc?

Reply:
Sorry, I was a bit confused as we previously dropped the mocks in this test, but you're right, the decompression is not a `parseEntity` matter anymore. 

My aim was to make sure we test the HLRC, irrespective of who in the implementation chain performs it (it is now the LLRC's responsibility but that might change).

 After looking at this a bit closer we are covered here by the `HighLevelRestClientCompressionIT` which makes sure the HLRC is tested.